CHANGES.rst

```diff
@@ -4,7 +4,9 @@
 4.1.3 (unreleased)
 ------------------
 
-- TBD
+- Fix _p_changed when removing items from small pure-Python
+  BTrees/TreeSets and when adding to Sets. See:
+  https://github.com/zopefoundation/BTrees/issues/13
```

Comment:
Issue 11 was about adding to the Tree versions; that didn't change here. This issue handles adding to the non-Tree Sets, as well as removing from Trees. I'm not sure how to reword the change entry to make it more clear, but I'm open to any suggestions.